### PR TITLE
[Neo Plugin RpcServer UT]Plugin rpcserver wallet UTs

### PR DESCRIPTION
**Pull request by Jim8y**
# Description

This pr adds unit tests for rpcserver wallet methods.  And also add comments to the rpc wallet methods 

Fixes # 

## Type of change

- [ ] Optimization (the change is only an optimization)
- [x] Style (the change is only a code style for better maintenance or standard purpose)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] TestOpenWallet
- [x] TestOpenInvalidWallet
- [x] TestDumpPrivKey
- [x] TestGetNewAddress
- [x] TestGetWalletBalance
- [x] TestGetWalletBalanceInvalidAsset
- [x] TestGetWalletUnclaimedGas
- [x] TestImportPrivKey
- [x] TestImportPrivKeyNoWallet
- [x] TestCalculateNetworkFee
- [x] TestCalculateNetworkFeeNoParam
- [x] TestListAddressNoWallet
- [x] TestListAddress
- [x] TestSendFromNoWallet
- [x] TestSendFrom
- [x] TestSendMany
- [x] TestSendToAddress
- [x] TestCloseWallet_WhenWalletNotOpen
- [x] TestDumpPrivKey_WhenWalletNotOpen
- [x] TestGetNewAddress_WhenWalletNotOpen
- [x] TestGetWalletBalance_WhenWalletNotOpen
- [x] TestGetWalletUnclaimedGas_WhenWalletNotOpen
- [x] TestImportPrivKey_WhenWalletNotOpen
- [x] TestCalculateNetworkFee_InvalidTransactionFormat
- [x] TestListAddress_WhenWalletNotOpen
- [x] TestCancelTransaction
- [x] TestInvokeContractVerify

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by cschuchardt88 — APPROVED** [review]

File: src/Plugins/RpcServer/Result.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static class Result
         /// <typeparam name="T">The return type</typeparam>
         /// <returns>The execution result</returns>
         /// <exception cref="RpcException">The Rpc exception</exception>
-        public static T Ok_Or<T>(this Func<T> function, RpcError err, bool withData = false)
+        public static T Ok_Or<T>(Func<T> function, RpcError err, bool withData = false)
```
Comment (shargon) [inline src/Plugins/RpcServer/Result.cs:26]: why remove this `this`?

Comment (Jim8y) [reply src/Plugins/RpcServer/Result.cs:26]: Cause they never and will not work, cannot extend func.

File: tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.Wallet.cs [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Plugins.RpcServer.Tests;
+
+partial class UT_RpcServer
+{
+
+    [TestMethod]
```
Comment (shargon) [inline tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.Wallet.cs:30]:
```suggestion
    [TestMethod]
```

**Review by shargon — DISMISSED** [review]

**Review by cschuchardt88 — APPROVED** [review]

File: tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.Wallet.cs [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+        snapshot.Commit();
+        return state.Hash;
+        // var engine = ApplicationEngine.Create(TriggerType.Application, null, snapshot);
+        // engine.LoadScript(new byte[] { 0x01 });
+        //
+        // engine.Get(new StorageContext
+        // {
+        //     Id = state.Id,
+        //     IsReadOnly = false
+        // }, new byte[] { 0x01 }).Value.Span.ToHexString().Should().Be(storageItem.Value.Span.ToHexString());
```
Comment (shargon) [inline tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.Wallet.cs:409]:
```suggestion
```

**Review by shargon — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Pull request merged**